### DISCUSSION TITLE
consult-ripgrep always grouping matches by file

### DISCUSSION THREAD

**Discussion by huangfeiyu** [minad/consult#1078] [Q&A] [answered]
I want the file name to be display as prefix of each match. But `consult-ripgrep ` always grouping matches by file. I don't change the value of `consult-ripgrep-args`, its value is "rg --null --line-buffered --color=never --max-columns=1000 --path-separator /   --smart-case --no-heading --with-filename --line-number --search-zip"
Here's a example of my search
![screenshot_2024-08-29_16-35-54](https://github.com/user-attachments/assets/e5366232-cb8e-4799-9b05-f729e3b2185c)

Thanks!

**Comment by huangfeiyu** [comment]
Hi @minad is grouping matches by file intentional?

**Comment by minad** [comment, marked as answer]
See https://github.com/minad/consult?tab=readme-ov-file#narrowing-and-grouping

**Comment by huangfeiyu** [comment]
Thank you @minad . The document is thorough, I should've read it carefully.
